coralproject/talk
# remove cruft by pulling out some immutable.js stuff

## What does this PR do?

I'm going to go through the dumb components and removing where we directly access immutable.js objects because it's just noise. I'm starting from the bottom up, so there will be several of these PRs. The goal is to convert the `mapStateToProps` to call `toJS()` at render time so that we don't have to call `comments.get('status')` or whatever every single time.

I've modified `ModerationQueue` and `CommentStream` in this PR.

## How do I test this PR?

- go to the admin
- you should still be able to see the comments in ModerationQueue and interact with them. No functionality change should have happened
